arc/src/main.c

```diff
@@ -322,6 +324,7 @@ static void handle_glcd(struct zjs_ipm_message* msg)
             glcd = device_get_binding(GROVE_LCD_NAME);
 
             if (!glcd) {
+                PRINT("failed to initialize Grove LCD\n");
```

Comment:
Yes, I forgot to add it to the glcd when I am implementing sensor, so I am thinking to include it as part of the patch, but I'll remove it and throw in another patch.